buildSrc/src/main/java/io/micronaut/guides/core/MacroUtils.java

```diff
@@ -44,7 +57,9 @@ public static List<String> addIncludes(@NonNull GuidesOption option,
             }
         } else {
             List<String> attributes = new ArrayList<>();
-            attributes.add("lines=" + licenseLoader.getNumberOfLines() + "..-1");
+            if (licenseLoader != null) {
+                attributes.add("lines=" + licenseLoader.getNumberOfLines() + "..-1");
+            }
```

Comment:
```suggestion
            if (licenseLoader.getNumberOfLines() > 0) {
                attributes.add("lines=" + licenseLoader.getNumberOfLines() + "..-1");
            }
```

Reply:
Not sure this is enough. It would still include it for rawTest macro for example, while it doesn't appear to have it